scripts/degenApe.js

```diff
@@ -115,301 +74,123 @@ const executeTx = async (calls, tx, fn, ...args) => {
+        console.log(`✔️ PickleJar deployed at: ${ txRefs.'jar'.address } `);
+
+        // Set Jar
+        txRefs.'setJarTx' = await executeTx(callAttempts, 'setJarTx', Controller.setJar, txRefs.'want', txRefs.'jar'.address);
+        console.log(`Jar Set!`);
+
+        // Approve Want
+        console.log(`Approving want token for deposit...`);
+        txRefs.'approveTx' = await executeTx(callAttempts, 'approveTx', txRefs.'wantContract'.approve, txRefs.'jar'.address, ethers.constants.MaxUint256);
+        console.log(`✔️ Successfully approved Jar to spend want`);
+      } else {
+        console.log(`Jar for this want already exists`);
+      }
 
       // Approve Strategy
-      await executeTx(callAttempts, 'approveStratTx', Controller.approveStrategy, txRefs['want'], txRefs['strategy'].address);
+      txRefs.'approveStratTx' = await executeTx(callAttempts, 'approveStratTx', Controller.approveStrategy, txRefs.'want', txRefs.'strategy'.address);
       console.log(`Strategy Approved!`);
 
-      // Set Jar
-      await executeTx(callAttempts, 'setJarTx', Controller.setJar, txRefs['want'], txRefs['jar'].address);
-      console.log(`Jar Set!`);
-
       // Set Strategy
-      await executeTx(callAttempts, 'setStratTx', Controller.setStrategy, txRefs['want'], txRefs['strategy'].address);
+      txRefs.'setStratTx' = await executeTx(callAttempts, 'setStratTx', Controller.setStrategy, txRefs.'want', txRefs.'strategy'.address);
       console.log(`Strategy Set!`);
       console.log(`✔️ Controller params all set!`);
 
       // Deposit Want
       console.log(`Depositing in Jar...`);
-      await executeTx(callAttempts, 'depositTx', txRefs['jar'].depositAll)
+      txRefs.'depositTx' = await executeTx(callAttempts, 'depositTx', txRefs.'jar'.depositAll)
       console.log(`✔️ Successfully deposited want in Jar`);
 
       // Call Earn
       console.log(`Calling earn...`);
-      await executeTx(callAttempts, 'earnTx', txRefs['jar'].earn);
+      txRefs.'earnTx' = await executeTx(callAttempts, 'earnTx', txRefs.'jar'.earn);
       console.log(`✔️ Successfully called earn`);
 
       //Push Strategy to be verified
-      testedStrategies.push(txRefs['strategy'].address)
+      testedStrategies.push(txRefs.'strategy'.address)
 
       // Call Harvest
-      console.log(`Waiting for ${sleepTime * 4 / 1000} seconds before harvesting...`);
-      await sleep(sleepTime * 4);
-      await executeTx(callAttempts, 'harvestTx', txRefs['strategy'].harvest);
-
-      await sleep(sleepTime, sleepToggle);
-      txRefs['ratio'] = await txRefs['jar'].getRatio();
-
-      if (txRefs['ratio'].gt(BigNumber.from(parseEther("1")))) {
-        console.log(`✔️ Harvest was successful, ending ratio of ${txRefs['ratio'].toString()} `);
+      console.log(`Waiting for ${ sleepConfigs.sleepTime * 4 / 1000 } seconds before harvesting...`);
```

Comment:
From what I can tell, these are the only references to sleepTime in the script? Why wouldn't you just 4x the value where it was declared rather than here? Seems weird to me.